projects/demo/src/modules/pipes/field-error/examples/1/index.ts

```diff
@@ -0,0 +1,32 @@
+import {Component} from '@angular/core';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+
+import {
+    passwordValidator,
+    superComputerValidator,
+} from '../../../../components/field-error/examples/1';
+
+@Component({
+    selector: 'tui-field-error-pipe-example-1',
+    templateUrl: './index.html',
+    changeDetection,
+    encapsulation,
+})
+export class TuiFieldErrorPipeExample1 {
+    readonly testValue1 = new FormControl('', [Validators.required, passwordValidator]);
+
+    readonly testValue3 = new FormControl('', [Validators.required, passwordValidator]);
+
+    readonly testForm = new FormGroup({
+        testValue1: this.testValue1,
+        testValue2: new FormControl('', [Validators.required, superComputerValidator]),
+    });
+
+    constructor() {
+        this.testValue1.valueChanges.subscribe(() => {
+            this.testValue1.markAsTouched();
+        });
```

Comment:
is it really necessary in the example?

Reply:
yes, the second example is about showing the error message immediately